internal/integrations/envsec/envsec.go

```diff
@@ -0,0 +1,63 @@
+package envsec
+
+import (
+	"encoding/json"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"go.jetpack.io/devbox/internal/boxcli/usererr"
+	"go.jetpack.io/devbox/internal/cmdutil"
+)
+
+func Env(projectDir string) (map[string]string, error) {
+
+	if err := ensureEnvsecInstalled(); err != nil {
+		return nil, err
+	}
+
+	if err := ensureEnvsecInitialized(); err != nil {
+		return nil, err
+	}
+
+	return envsecList(projectDir)
+}
+
+func ensureEnvsecInstalled() error {
+	if !cmdutil.Exists("envsec") {
+		return usererr.New("envsec is not installed or not in path")
+	}
+	return nil
+}
+
+func ensureEnvsecInitialized() error {
+	cmd := exec.Command("envsec", "init")
+	// TODO handle user not logged in
+	// envsec init is currently broken in that it exits with 0 even if the user is not logged in
+	return cmd.Run()
+}
+
+func envsecList(projectDir string) (map[string]string, error) {
+	cmd := exec.Command(
+		"envsec", "ls", "--show",
+		"--format", "json",
```

Comment:
I'm not sure if we should do this. I worry that the secrets printed can be discovered through some proc file somewhere. 

Can we just use the envsec go library directly?

Reply:
I was under impression we wanted to do CLI integration in order to avoid bloating devbox. This has the benefit of allowing other 3rd parties to integrate as well if we provide a standard interface.

If stderr and stdout are captured in a buffer, is there really any risk the outputs ends up anywhere else?

We can also go the RPC route, but that's more work and feels a bit overkill.

Reply:
Looking deeper it might be ok based on [this line](https://cs.opensource.google/go/go/+/master:src/os/exec/exec.go;l=984). So I guess we are good